src/utils/localized.js

```diff
@@ -58,6 +58,10 @@ export const createLocalizedDataAttributes = (props, language) =>
 export const getHasErrorOnRemainingLanguages = (errors, selectedLanguage) =>
   Boolean(errors) && without(Object.keys(errors), selectedLanguage).length > 0;
 
+export const getHasWarningOnRemainingLanguages = (warnings, selectedLanguage) =>
```

Comment:
As far as I can tell, these two functions are exactly the same.

We could also rename one to `getHasIssueOnRemainingLanguages` and then either pass in the warnings or the errors. Then we avoid the duplication.

Reply:
IMO, having `getHasIssueOnRemainingLanguages` is too generic and harder to understand the abstraction. Having them separate helps to understand it better. In the end, I'd prefer to duplicate if its in favour of keeping the code understandable

Reply:
Would a code comment help instead?

Reply:
I'd keep the code separated. If something changes in the warning/error validation, its better to have them as is.

Reply:
👍